commitserver/commit/util_test.go

```diff
@@ -0,0 +1,39 @@
+package commit
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestMakeSecureTempDir(t *testing.T) {
+	tempDir, cleanup, err := makeSecureTempDir()	
+	
+	if err != nil {
+		t.Fatalf("Failed to create secure temporary directory: %v", err)
+	}
```

Comment:
Instead of `t.Fatalf`, could you switch to `require.NoError`?

Reply:
Done with requests!